lib/challenge.js

```diff
@@ -48,10 +92,17 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
-
 // Answer Starts Here
 
+var some_pet = {
+  name: 'maggie',
+  species: 'cat',
+  breed: 'tiger''
+  noise: function() {
+    console.log('meow');
+};
```

Comment:
The curly brace on this line closes the function body; the object literal remains unclosed. The semicolon, then, is a syntax error. The semicolon on the line above is not, though. Can you explain why? Partial credit.